validate/calculate_coherence_distinct.ipynb

```diff
@@ -1,928 +1,901 @@
-        "id": "E4xRaHxflvMq"
-      },
-      "id": "E4xRaHxflvMq",
-      "execution_count": 101,
-      "outputs": []
-    },
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "plt.plot(years,[dem_1970_cohere,dem_1980_cohere,dem_1990_cohere,dem_2000_cohere],'--', label='Organizational Ecology', color = 'green')\n",
+    "\n",
+    "plt.plot(years,[cult_1970_cohere,cult_1980_cohere,cult_1990_cohere,cult_2000_cohere], label='Organizational Institutionalism', color = 'red')\n",
+    "\n",
+    "plt.plot(years,[rela_1970_cohere,rela_1980_cohere,rela_1990_cohere,rela_2000_cohere],':', label='Resource Dependence', color = 'blue')\n",
+    "\n",
+    "plt.xlabel('Time period')\n",
+    "plt.ylabel('Coherence')\n",
+    "\n",
+    "# plt.title('Coherence scores vs. decades')\n",
+    "ax = plt.subplot(111)\n",
+    "ax.legend(loc='upper center', bbox_to_anchor=(0.5, 1.2),\n",
+    "          ncol=3, frameon=False)\n",
+    "\n",
+    "\n",
+    "# plt.legend();\n",
+    "plt.ylim(0.15, 0.35)\n",
+    "plt.yticks(np.linspace(0.15, 0.35, 5))\n",
+    "plt.grid()\n",
+    "filepath =  'coherence_score_core_dict' + \".png\"\n",
+    "\n",
+    "plt.savefig(filepath, bbox_inches='tight', dpi= 2000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "XQJnOjxGx73g",
+   "metadata": {
+    "id": "XQJnOjxGx73g"
+   },
+   "source": [
+    "### Calculate distinctiveness scores of each dictionary "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "4uz0gncQkhO9",
+   "metadata": {
+    "id": "4uz0gncQkhO9"
+   },
+   "outputs": [],
+   "source": [
+    "def dict_distinct(dict1, dict2, wem_model):\n",
+    "    '''Computes the average cosine distance score of terms in dict1 with all terms in dict2,\n",
+    "    effectively measuring the opposition/non-coherence between the two dictionaries.\n",
+    "    \n",
+    "    NOTE: For an unknown reason, calling this function deletes terms from thisdict.\n",
+    "    \n",
+    "    Inputs: List of key terms, word2vec model.\n",
+    "    Output: Average cosine distance score of each word in dict1 with all words in dict2.'''\n",
+    "    \n",
+    "    # Initialize average distance variables:\n",
+    "    # word_avg_dist = 0\n",
+    "    # word_avg_dists = 0\n",
+    "    # dicts_avg_dist = 0\n",
+    "    \n",
+    "    # # Compute average cosine similarity score of each term in dict1 with all terms in dict2:\n",
+    "    # for word in dict1:\n",
+    "    #     word_avg_dist = (wem_model.distances(word, other_words=dict2).sum())/len(dict2) # Total diffs of word with all other words, take average\n",
+    "    #     word_avg_dists += word_avg_dist # Add up each average distance, incrementally\n",
+    "    # dicts_avg_dist = word_avg_dists/len(dict1) # Find average cosine distance score by dividing sum by # words\n",
+    "    \n",
+    "    \n",
```

Comment:
```suggestion
```

Reply:
No need for this commented out code here, which you use in the different function for dict coherence.